_episodes/01-numpy.md

```diff
@@ -29,34 +29,68 @@ keypoints:
 - "Use the `pyplot` library from `matplotlib` for creating simple visualizations."
 ---
 
-In this lesson we will learn how to manipulate the inflammation dataset with Python. Before we
-discuss how to deal with many data points, we will show how to store a single value on the computer.
+In this lesson we will learn how to work with arthritis inflammation datasets in Python. However,
+before we discuss how to deal with many data points, let's learn how to work with single data values.
 
-You can get output from python by typing math into the console:
+## Variables
+
+Any Python interpreter can be used as a calculator:
+~~~
+3 + 5 * 4
+~~~
+{: .language-python}
 ~~~
-3+5
-12/7
+23
 ~~~
+{: .output}
 
-However, to do anything useful and/or interesting we need to assign values to _variables_
-(or link _objects_ to names/variables).
-The line below [assigns]({{ page.root }}/reference/#assign) the value `60` to a
-[variable]({{ page.root }}/reference/#variable) `weight_kg`:
+This is great but not very interesting.
+To do anything useful with data, we need to assign its value to a _variable_.
+In Python, we can [assign]({{ page.root }}/reference/#assign) a value to a
+[variable]({{ page.root }}/reference/#variable), using the equals sign `=`.
+For example, to assign value `60` to a variable `weight_kg`, we would execute:
 
 ~~~
 weight_kg = 60
 ~~~
 {: .language-python}
 
-A variable is a name for a value,
-such as `x_val`, `current_temperature`, or `subject_id`.
-Python's variables must begin with a letter and are
-[case sensitive]({{ page.root }}/reference/#case-sensitive).
-We can create a new variable by assigning a value to it using `=`.
-When we are finished typing and press <kbd>Shift</kbd>+<kbd>Return</kbd>,
-the notebook runs our command.
+From now on, whenever we use `weight_kg`, Python will substitute the value we assigned to
+it. In essence, **a variable is just a name for a value**.
+
+In Python, variable names:
+
+ - must begin with a letter, and
+ - are [case sensitive]({{ page.root }}/reference/#case-sensitive).
 
-Once a variable has a value, we can print it to the screen:
+This means that, for example:
+ - `weight0` is a valid variable name, whereas `0weight` is not
+ - `weight` and `Weight` are different variables
+
+## Types of data
+Python knows various types of data. The most common ones are:
```

Comment:
Are these "the most common"? It could say "Three common ones".

Reply:
Sounds good